src/transformers/modeling_flax_pytorch_utils.py

```diff
@@ -109,8 +109,20 @@ def is_key_or_prefix_key_in_dict(key: Tuple[str]) -> bool:
 
 
 def convert_pytorch_state_dict_to_flax(pt_state_dict, flax_model):
+    try:
```

Comment:
sounds good to me!

Reply:
Is this necessary? Looks to me to be redundant! We check for `torch` import in `load_pytorch_checkpoint_in_flax_state_dict`:
https://github.com/huggingface/transformers/blob/d0d5aee1dd26de108a901921b9df19b889430645/src/transformers/modeling_flax_pytorch_utils.py#L45-L53
And then call `convert_pytorch_state_dict_to_flax` 9 lines later:
https://github.com/huggingface/transformers/blob/d0d5aee1dd26de108a901921b9df19b889430645/src/transformers/modeling_flax_pytorch_utils.py#L62
So this check in `convert_pytorch_state_dict_to_flax`is superseded by the parent function `load_pytorch_checkpoint_in_flax_state_dict`.

Reply:
Yeah let's remove it - I just had to add `# noqa: F821` to make the quality test pass here